fastchat/serve/shutdown_serve_by_shell.py

```diff
@@ -0,0 +1,31 @@
+"""
+Usage：
+python llm_api_shutdown.py --serve all
```

Comment:
docstring not consistent with the filename?
also maybe try a shorter name like `shutdown_server.py`?

Reply:
done.